mopidy/models/serialize.py

```diff
@@ -2,7 +2,9 @@
 
 import json
 
-from mopidy.models.immutable import ImmutableObject
+from mopidy.models import immutable
+
+_MODELS = ['Ref', 'Artist', 'Album', 'Track', 'TlTrack', 'Playlist']
```

Comment:
I agree we don't need to be able to deserialize SearchResult. What about the custom models in -Podcast, etc? They're never passed to the web client and back?